modules/account-lib/test/unit/coin/algo/utils.ts

```diff
@@ -229,4 +231,36 @@ describe('utils', () => {
     should.equal(isValid, true);
     should.equal(isValidSecretKey, true);
   });
+
```

Comment:
Please add more tests. Take into account that you have useful data here https://github.com/BitGo/BitGoJS/blob/feat/algo-tokens/modules/account-lib/test/resources/algo/index.ts, you can use these accounts to validate keys, verify signatures, etc.

Reply:
You should cover the 'failing cases' for the new methods you added